ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -187,6 +187,8 @@ objects:
       env:
       - name: TYPE
         value: ${CLUSTER_TYPE}
+      - name: NETWORK_TYPE
```

Comment:
CLUSTER_TYPE is a template variable and therefore probably you can change that. this would need to be something similar.

Reply:
Sorry but I don't follow what CLUSTER_TYPE has to do here.
I'm adding a new env variable, just so the networkType can be configured, instead of using the current default which is OpenShiftSDN.

CLUSTER_TYPE is a var to signal where should be the cluster be created, whether GCP or AWS.

Reply:
CLUSTER_TYPE is supposed to describe the type of cluster (a profile), not just the platform.  "aws-network-multitenant" would thus be a valid cluster type.

However, at this point you can't easily combine those (aws + multitenant), so this is an acceptable hack.

However, since you don't use NETWORK_TYPE anywhere you shouldn't abstract it.  Please remove this reference and simply leave the network manifest.